test/e2e/go.mod

```diff
@@ -0,0 +1,79 @@
+module test
+
+go 1.18
+
+require github.com/kudobuilder/kuttl v0.12.1
+
+require (
```

Comment:
This, `go.sum` and `tools.go` can be deleted as well, now that we install kuttl with `go install`